pep-0012.rst

```diff
@@ -7,7 +7,7 @@ Status: Active
 Type: Process
 Content-Type: text/x-rst
 Created: 05-Aug-2002
-Post-History: 30-Aug-2002
+Post-History: `30-Aug-2002 <https://mail.python.org/archives/list/python-dev@python.org/thread/KX3AS7QAY26QH3WIUAEOCCNXQ4V2TGGV/>`_
```

Comment:
```suggestion
Post-History: `30-Aug-2002 <https://mail.python.org/archives/list/python-dev@python.org/thread/KX3AS7QAY26QH3WIUAEOCCNXQ4V2TGGV/>`__
```

Reply:
Oops, my mistake. Rebased in and fixed, thanks!